webmails/roundcube/config/config.inc.php

```diff
@@ -28,7 +28,7 @@
 $config['default_port'] = '10143';
 
 // Sieve script management
-$config['managesieve_host'] = '{{ IMAP_ADDRESS or "imap" }}';
+$config['managesieve_host'] = '{% if TLS_FLAVOR != "notls" %}ssl://{% endif %}{{ HOSTNAMES.split(",")[0] }}:4190';
```

Comment:
What do we gain by connection through the public address here?
imap and smtp is done through front (unencrypted), so we could also du sieve through front unencrypted (port 14190)
Another (maybe better) way would be to do everything the webmails do through the public and encrypted endpoints like every client does.

Reply:
It bypasses the rate-limits and authentication is handled differently (with temporary tokens). Using the public ports would mean allowing that type of tokens on those ports and finding another way to rate-limit/exempt from rate limits.

Reply:
OK. Let's keep it simple.